store/sql_team_store.go

```diff
@@ -496,7 +496,13 @@ func (s SqlTeamStore) SaveMember(member *model.TeamMember) StoreChannel {
 				TeamMembers
 			WHERE
 				TeamId = :TeamId
-				AND DeleteAt = 0`, map[string]interface{}{"TeamId": member.TeamId}); err != nil {
+				AND DeleteAt = 0
```

Comment:
This nested query is a performance pedelty over the previous query. This feature is only meant for stopping rouge bots from creating many users. Are we sure this is worth it?